results/cmd/watcher/main.go

```diff
@@ -54,25 +58,23 @@ func main() {
 	}
 	defer conn.Close()
 	client := pb.NewResultsClient(conn)
-
 	sharedmain.MainWithContext(injection.WithNamespaceScope(signals.NewContext(), *namespace), "watcher", func(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
 		logger := logging.FromContext(ctx)
 		taskRunInformer := taskruninformer.Get(ctx)
-
 		c := &reconciler{
 			logger:        logger,
 			taskRunLister: taskRunInformer.Lister(),
 			client:        client,
 		}
-		impl := controller.NewImpl(c, c.logger, pipeline.PipelineRunControllerName)
+		impl := controller.NewImpl(c, c.logger, pipeline.TaskRunControllerName)
```

Comment:
These changes appear to be unrelated to adding support for `Update`. Let's break this out to it's own PR.

Reply:
Sure